.github/workflows/trivy_check.yaml

```
@@ -59,4 +59,4 @@ jobs:
 
       - name: Run Trivy vulnerability scanner in fs mode
         run: |
-          trivy fs . --quiet --config trivy.yaml
+          trivy fs . --quiet --config trivy.yaml
```

```suggestion
          trivy fs . --quiet --config trivy.yaml

```
Looks like nothing changed except removing line break